cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -180,7 +180,7 @@ void SparsePoseGraph::ComputeConstraint(const mapping::NodeId& node_id,
         submap_id, submap_data_.at(submap_id).submap.get(), node_id,
         trajectory_nodes_.at(node_id).constant_data.get());
   } else {
-    if (connected_components_.TransitivelyConnected(
+    if (trajectory_connectivity_state_.TransitivelyConnected(
```

Comment:
Implementing `TrajectoryConnectivityState` and using it could have become two separate PRs. We are not really making use of the new functionality here anyway.

Reply:
Noted for the future.